doc/cookbook/source/examples/classifier/lmnn.rst

```diff
@@ -0,0 +1,47 @@
+=================================
+Largest Margin Nearest Neighbours
+=================================
+
+Largest Margin Nearest Neighbours is a metric learning algorithm.  It learns a metric that can be used with the K-Nearest neigbour algorithm.
+
+The Mahalanobis distance metric which is obtained is computed for  samples :math:`x_i` and :math:`x_j` as:
+
+.. math::
+
+	( (\vec{x_i} - \vec{x_j})^T \bf{M} (\vec{x_i} - \vec{x_j}) )
+   
+Here :math:`\bf{M}` is the Mahalanobis matrix.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` as
+
+.. sgexample:: lmnn.sg:create_features
+
+We create an instance of :sgclass:`CLMNN` with parameter k (nearest neighbours).
```

Comment:
dont mention parameter names in the text (they might change),
just say that you provide the number of neighbours as parameter